keras_nlp/layers/sine_positional_embedding_test.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for Sinusoidal Positional embedding."""
+
+
+import tensorflow as tf
+from tensorflow import keras
+
+from keras_nlp.layers import sine_positional_embedding
+
+
+class SinePositionEncodingTest(tf.test.TestCase):
+    def test_valid_call(self):
```

Comment:
Maybe add a test if there are multiple dims before sequence length? The batch of sequence of sequences of features case. Don't think this will be common, but we should make sure it works.